tests/test_watcher_enhanced.py

```
@@ -0,0 +1,439 @@
+
+        assert watcher_instance.email_monitor_status == "Polling"
+        assert watcher_instance.email_last_check_time == 123456.0
+        assert watcher_instance.email_jobs_found_session == 5
+
+
+class TestRawWSMessages:
+    """Test raw WebSocket message capture."""
+
+    def test_capture_raw_ws_message_when_enabled(self, watcher_instance, mock_config):
+        """Test capturing raw WS messages when debug is enabled."""
+        mock_config.get.side_effect = lambda s, k: True if k == "raw" else None
```

Ruff (0.14.14)

Unused lambda argument: `s`

(ARG005)